<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>601</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff55ff;&quot;&gt;Trần Đức Trung Kiên&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="changeTextpushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fontSizepushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Size ++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alignLeftpushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Align Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alignCenterpushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Align Center</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alignRightpushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Align Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showPNGpushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show .PNG format</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showGIFpushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show .GIF format</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/kien0/OneDrive - zxcvb/Pictures/ThatTinh1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
